recipes/uncertainpy/meta.yaml

```
@@ -0,0 +1,72 @@
+{% set name = "uncertainpy" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1df39206390f1410fccdb4d6f6c611203ded46d6fe176e17593a207c1b42a2e6
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - chaospy
+    - h5py
+    - matplotlib >=2
+    - multiprocess
+    - numpy
+    - python
+    - salib
+    - scipy
+    - seaborn
+    - six
+    - tqdm
+    - xvfbwrapper
+
+test:
+  imports:
+    - uncertainpy
+    - uncertainpy.core
+    - uncertainpy.features
+    - uncertainpy.models
+    - uncertainpy.plotting
+    - uncertainpy.plotting.prettyplot
+    - uncertainpy.utils
+
+about:
+  home: https://github.com/simetenn/uncertainpy
+  license: GNU GPLv3
```

From https://spdx.org/licenses/:
```suggestion
  license: GPL-3.0
```

---

I didn't know about it... conda skeleton generated this way above. Should I relate it?